### PR TITLE
Feature/tri 1576 concept handling of multiple dtr assets

### PR DESCRIPTION
**Pull request by ds-jhartmann**

### CODE REVIEW

File: docs/concept/Discovery-process-DTR.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+````mermaid
+sequenceDiagram
+    IRS ->> DTR: /query for globalAssetId
+    DTR ->> IRS: return list of two results
+    IRS ->> IRS: use first
+````
+
+## To be clarified
+
+1. How should the procedure look like for Notifications?
```
[the remark this answers is not in the document]
Comment (ds-jhartmann) [reply docs/concept/Discovery-process-DTR.md]: Created PBI https://github.com/eclipse-tractusx/item-relationship-service/issues/235

[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/Discovery-process-DTR.md]: wfm

File: docs/concept/Discovery-process-DTR.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+| EDC          | Eclipse Dataspace Connector |
+| DTR          | Digital Twin Registry       |
+| DT           | Digital Twin                |
+
+## Scenarios
+
+IRS is not able to validate if a certain data offer is correct or outdated. IRS will take the first responding offer
+which is available.
+
+### EDC with multiple DTRs
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/Discovery-process-DTR.md:20]: wfm

**Review by mkanal — APPROVED** [review]

**Comment by jzbmw** [comment]
approved and read

**Pull request merged**